src/metabase/models/cloud_migration.clj

```diff
@@ -33,29 +33,42 @@
 (t2/deftransforms :model/CloudMigration
   {:state mi/transform-keyword})
 
-(defsetting migration-store-url
-  (deferred-tru "Store URL for migrations. Internal test use only.")
+(defsetting store-use-staging
+  (deferred-tru "If staging store should be used instead of prod. True on dev.")
+  :type       :boolean
   :visibility :internal
-  :default    (if config/is-dev?
-               "https://store-api.staging.metabase.com/api/v2/migration"
-               "https://store-api.metabase.com/api/v2/migration")
+  :default    config/is-dev?
+  :doc        false
+  :export?    false)
+
+(defsetting store-url
+  (deferred-tru "Store URL.")
+  :visibility :admin ;; should be :internal, but FE doesn't get internal settings
```

Comment:
This setting should be `:internal` but the FE doesn't get internal settings. I could make it get them, but I'm not confident that changing visibility logic is the better approach here. Defaulted to the low effort approach.

@dpsutton WDYT?

Reply:
It seems like `:admin` is what you want, given the FE needs read the setting, but why do you think it should be `:internal`?

Reply:
I'm trying to tell the FE what the store url is, in a way that I can alter it both on local development environments and when testing jar/docker artifacts.

Reply:
I think `:admin` is expected here if it needs to be surfaced to the FE. It means that it'll only be surfaced to admin users and not non-admins.

If you want to ensure it's not _set_ by admins, set the `:setter` to `:none`.